code/__DEFINES/atom_hud.dm

```diff
@@ -55,3 +55,5 @@
 #define NOTIFY_JUMP "jump"
 #define NOTIFY_ATTACK "attack"
 #define NOTIFY_ORBIT "orbit"
+
+#define ADD_HUD_TO_COOLDOWN 20 //cooldown for being shown the images for any particular data hud
```

Comment:
Side note that this must remain shorter than the second value of https://github.com/tgstation/tgstation/blob/master/code/controllers/subsystem/garbage.dm#L9